easybuild/easyconfigs/r/Ruby-FPM/Ruby-FPM-2.1.5-1.0.0-foss-2015a.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'Bundle'
+
+name = 'Ruby-FPM'
```

Comment:
We're not doing this for other stuff, everything is organised based on software name.

For example, the HPCBIOS bundles aren't grouped together either, they each have their own subdirectory.

I'm not strictly opposing grouping things together, but note that this will complicate things for `--robot`. How can it know if should be looking for `Ruby-FPM` or `FPM` under the `Ruby` or `Ruby-bundles` directory?

Actually, I don't think there's a need for a bundle at all here... Why not simply include all these extensions (except for FPM itself) in the Ruby easyconfig, like we do for Python, Perl, R? It probably makes sense to keep a separate easyconfig file for FPM itself, and install that using `easyblock = 'RubyGem'`...

The bundle approach is mostly intended to handle situations where extra dependencies other than the 'mother' language are involved (cfr. #1387 for an example involving R).